build/rollup.config.build.js

```diff
@@ -8,24 +8,48 @@ const plugins = [
   babel(),
   replace({
     "process.env.NODE_ENV": JSON.stringify("production")
-  }),
+  })
 ];
 
+const externalModules = [
+  "prosemirror-example-setup",
+  "prosemirror-history",
+  "prosemirror-keymap",
+  "prosemirror-menu",
+  "prosemirror-model",
+  "prosemirror-schema-basic",
+  "prosemirror-state",
+  "prosemirror-test-builder",
+  "prosemirror-view",
+  "preact",
+  "prosemirror-tables",
+  "prosemirror-utils",
+  "uuid",
+  "prop-types",
+  "react-is",
+  "hoist-non-react-statics"
+];
+
+const external = id =>
+  /(lodash)/.test(id) || externalModules.includes(id);
+
 export default [
   {
     input: "src/ts/index.ts",
     output: {
       file: "dist/index.js",
       format: "cjs"
     },
-    plugins
+    plugins,
+    external
```

Comment:
By adding the rollup `commonjs` plugin to our bundle, the plugins listed in `external` are now eligible for bundling. (Previously, they were ignored). We only want to bundle material-ui.

We could look to automate this with something like [`rollup-plugin-peer-deps-external`](https://www.npmjs.com/package/rollup-plugin-peer-deps-external) in future, although arguably it's not a bad thing to be explicit in a library.